.travis.yml

```diff
@@ -36,82 +44,148 @@ jobs:
             - g++-6
       install: *ccache-setup-steps
       script:
-        - pyenv global ${PYTHON_VERSION}
+        - pyenv global ${PYTHON3_VERSION}
+        # - ./configure  # workaround pending #25878
+        - python3 configure.py
+        - make -j2 -C out V=1 v8
+
+    - name: "Compile Node.js (py2)"
+      cache:
+        ccache: true
+        directories:
+          - $HOME/.ccache/py${PYTHON2_VERSION}
+
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-6
+      install: *ccache-setup-steps
+      script:
+        - pyenv global ${PYTHON2_VERSION}
         - ./configure
         - make -j2 V=1
-        - cp out/Release/node /home/travis/.ccache
-        - cp out/Release/cctest /home/travis/.ccache
+        - mkdir -p $HOME/.ccache/py${PYTHON2_VERSION}
+        - cp out/Release/node $HOME/.ccache/py${PYTHON2_VERSION}
+        - cp out/Release/cctest $HOME/.ccache/py${PYTHON2_VERSION}
+        - ls -lr $HOME/.ccache
```

Comment:
Just curious, is this for potential debugging?

Reply:
Good eyes.  Removed.  Thanks.